store-tests

```diff
@@ -82,7 +82,7 @@ def main():
     # 'repo', 'url', 'state', 'description'
     # 'started' - when this run was picked by bot
     # 'posted' - when this run was triggered
-    # 'finished' - when this run finished - 'state' is then either 'success', 'failure' or 'error'
+    # 'finished' - when this run finished - 'state' is then either 'pending', 'success', 'failure' or 'error'
```

Comment:
That comment is a bit confusing -- The "finished" field does not exist for state "pending"; move the comment to "posted"?

Reply:
Oh right, I just saw listing of states so I added it without reading what it is really about. Removed it.